src/MultiOwnable.sol

```diff
@@ -82,31 +92,47 @@ contract MultiOwnable {
     /// @notice Convenience function to add a new owner address.
     ///
     /// @param owner The owner address.
-    function addOwnerAddress(address owner) public virtual onlyOwner {
+    function addOwnerAddress(address owner) external virtual onlyOwner {
```

Comment:
other clean up I am grouping in here

Reply:
I'm wondering if this is overly restrictive. I could imagine a plugin/module that might help users manage owners. Depending on how the community agrees on module/plugin support, this call could be executed in the context of  this contract.

Reply:
Fair point

Reply:
Wt now? DM me pls